### PR TITLE
newly updated nodemon website with react

### PR DESCRIPTION
**Pull request by ayushkavad**
newly updated nodemon website

### CODE REVIEW

**Comment by ayushkavad** [comment]
this is newly updated nodemon website with documentation build with React.
nodemon

**Comment by ayushkavad** [comment]
nodemon2

**Comment by ayushkavad** [comment]
nodemon3

**Comment by remy** [comment]
Sorry chap, but the homepage is a single static html file which is a million times better than these new 37 files. Closing for that reason.

File: client/src/index.js
Comment (0123sandeepkumar) [inline client/src/index.js:1]: what is component doing

**Pull request closed without merging**